scripts/travis/.travis.yml

```diff
@@ -25,6 +25,7 @@ cache:
   - "$HOME/.drush/cache"
   - "$HOME/.npm"
   - "$HOME/.nvm"
+  - "vendor"
```

Comment:
Why add this back?

Reply:
I was just reverting the changes associated with php7, if this wasn't meant to be part of that commit (or was two commits in one), then this should be taken out of the PR.

Reply:
Let's not add it back.